### ISSUE TITLE
machinery config --help should mention `KEY=VALUE` syntax

### ISSUE THREAD

**Issue by tgoettlicher** [SUSE/machinery#1058]
`machinery config --help` prints:
```
NAME
    config - Show or change machinery's configuration

SYNOPSIS
    machinery [global options] config [KEY] [VALUE]

DESCRIPTION
    Show or change machinery's configuration.

    The value of a key is shown when no value argument is passed. If neither
    the key argument nor the value argument are specified a list of all keys and
    their values are shown. 
```

It should also mention that the syntax `KEY=VALUE` is also valid.

**Comment by ajaeger** [comment]
Todo: Add KEY=VALUE to help output.

**Comment by keberle94** [comment]
Fixed by #1201
